meds/meds.py

```diff
@@ -1042,24 +1030,11 @@ def _make_composite_image(
         return cim
 
     def _get_extension_name(self, type):
-        if type == 'image':
-            return "image_cutouts"
-        elif type == "weight":
-            return "weight_cutouts"
-        elif type == "seg":
-            return "seg_cutouts"
-        elif type == "bmask":
-            return "bmask_cutouts"
-        elif type == "ormask":
-            return "ormask_cutouts"
-        elif type == "noise":
-            return "noise_cutouts"
-        else:
-            ext = "%s_cutouts" % type
+        ext = "%s_cutouts" % type
+        if ext not in self._fits:
             if ext not in self._fits:
                 raise ValueError("bad cutout type '%s'" % type)
```

Comment:
```suggestion
            raise ValueError("bad cutout type '%s'" % type)
```

Why the extra if statement here?